File: Bmi_2/app/src/main/java/com/pjatk/bmi_2/model/Question.java
package com.pjatk.bmi_2.model;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

import java.util.ArrayList;
import java.util.Collections;

public class Question {

    private String _questionText;
    private ArrayList<Answer> _answers = new ArrayList<>();

    public  Question (JSONObject question) throws JSONException {
        _questionText = question.getString("question");
        JSONArray answersArray = question.getJSONArray("answers");
        for(int i = 0; i < answersArray.length(); i++){
            JSONObject object = answersArray.getJSONObject(i);
            _answers.add(new Answer(object.getString("text"),object.getBoolean("isCorrect")));
        }

        Collections.shuffle(_answers);
    }

    public String getQuestionText(){
        return _questionText;
    }

    public ArrayList<Answer> getAnswers(){
        return _answers;
    }
}

File: BMICalculator/app/src/main/java/com/szymon/bmicalculator/model/Question.kt
package com.szymon.bmicalculator.model

import org.json.JSONException
import org.json.JSONObject
import java.util.*

class Question(question: JSONObject) {
    private var _questionText: String = ""
    private val _answers = ArrayList<Answer>()

    init{
        _questionText = question.getString("question")
        val answersArray = question.getJSONArray("answers")
        for (i in 0 until answersArray.length()) {
            val `object` = answersArray.getJSONObject(i)
            _answers.add(Answer(`object`.getString("text"), `object`.getBoolean("isCorrect")))
        }

        _answers.shuffle()
    }


    fun getQuestionText(): String {
        return _questionText
    }

    fun getAnswers(): ArrayList<Answer> {
        return _answers
    }
}